packages/docs/src/routes/docs/advanced/dollar/index.mdx

```diff
@@ -260,13 +260,13 @@ export const onScroll_1 = () => alert('scroll');
 
 Notice:
 
-1. All that the developer had to do was to wrap the function in the `$()` to signal to the Optimizer that the function should be moved to a new file and therefore lazy-loaded.
-2. The `onScroll` had to be implemented slightly differently as it needs to take into account the fact that the `QRL` of the function needs to be loaded before it can be used. In practice using `qImport` is rare in Qwik applications as the Qwik framework provides higher-level APIs that rarely expect the developer to work with `qImport` directly.
+1. All that the developer had to do was to wrap the function in the `$()` to signal to the Optimizer that the function should be moved to a new file and therefore be lazy-loaded.
+2. The `onScroll` function had to be implemented slightly differently as it needs to take into account the fact that the `QRL` of the function needs to be loaded before it can be used. In practice using `QRL.resolve()` is rare in Qwik applications as the Qwik framework provides higher-level APIs that rarely expect the developer to work with `QRL.resolve()` directly.
 
-However, wrapping code in `$()` is a bit inconvenient. For this reason, Optimizer implicitly wraps the first argument of any function call, which ends with `$`. (Additionally, one can use `implicit$FirstArg()` to automatically perform the wrapping and type matching of the function taking the `QRL`.)
+However, wrapping code in `$()` is a bit inconvenient. For this reason, one can use `implicit$FirstArg()` to automatically perform the wrapping and type matching of the function taking the `QRL`. The function passed into `implicit$FirstArg()` should have a suffix of `Qrl`, and the result of that function should be set to a value with a suffix of `$`;
 
 ```tsx
-const onScroll$ = implicit$FirstArg(onScroll);
+const onScroll$ = implicit$FirstArg(onScrollQrl); 
```

Comment:
Shouldn't this be `onScroll` instead of `onScrollQrl`. `onScrollQrl` isn't mentioned anywhere else.